inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_non_max_suppression_opset3.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
The same questions as for unit tests - I think they can be generalized and combined + using info from targeted network can be useful if it is not here already.
Also, if some of them do not require StaticShapeNonMaxSuppresion - they can be enabled now..